src/org/ods/orchestration/usecase/LeVADocumentUseCase.groovy

```diff
@@ -1107,22 +1132,25 @@ class LeVADocumentUseCase extends DocGenUseCase {
         def watermarkText = this.getWatermarkText(documentType, this.project.hasWipJiraIssues())
 
         if (!repo.data.openshift && repo.data.odsBuildArtifacts) {
-            repo.data["openshift"] = [:]
-            repo.data.openshift << repo.data.odsBuildArtifacts.subMap (["builds","deployments"])
+            repo.data['openshift'] = [:]
+            repo.data.openshift << repo.data.odsBuildArtifacts.subMap (['builds','deployments'])
             this.steps.echo("fetched openshift data from build for repo: ${repo.id} \r${repo.data.openshift}")
         }
 
-        def deploynoteData = "Components were built & deployed during installation."
-        if (!repo.data.openshift?.builds || repo.data.openshift?.builds?.size() == 0) {
-            deploynoteData = "NO Components were built during installation, existing components (created in Dev) were deployed."
+        def deploynoteData = 'Components were built & deployed during installation.'
+        if (repo.data.odsBuildArtifacts?.resurrected) {
+            deploynoteData = 'Components were found up to date with version control -no deployments happend!\r' +
+                ' SCRR was restored from the corresponding creation build'
+        } else if (!repo.data.openshift?.builds || repo.data.openshift?.builds?.size() == 0) {
```

Comment:
`repo.data.openshift?.builds` is already "falsy" in Groovy when `size() == 0`, so you shouldn't need the `||` here.